test/unit/model/define.test.js

```diff
@@ -30,20 +30,12 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       expect(Model.rawAttributes).not.to.have.property('updated_at');
     });
 
-    it('should throw when id is added but not marked as PK', () => {
```

Comment:
this test should not have been removed. It's not incompatible with `noPrimaryKey`